custom_components/ha-glocaltokens/manifest.json

```diff
@@ -1,10 +1,10 @@
 {
   "domain": "ha-glocaltokens",
-  "name": "HA-Glocaltokens",
+  "name": "HA-Glocaltokens)",
   "documentation": "https://github.com/leikoilja/ha-glocaltokens",
   "issue_tracker": "https://github.com/leikoilja/ha-glocaltokens/issues",
   "dependencies": [],
   "config_flow": true,
   "codeowners": ["@leikoilja"],
-  "requirements": ["glocaltokens==0.1.4"]
+  "requirements": ["glocaltokens==0.1.4","requests==2.23.0","urllib3==1.25.11"]
```

Comment:
`requests` and `urllib3`are already specified in the HA requirements, so we don't need to specify those here. That field is just for dependencies needed specifically for the integration, such as `glocaltokens`

Reply:
Why do we even need urllib?